server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/metrics/GeoBoundsIT.java

```diff
@@ -69,25 +70,28 @@ public void testSingleValuedField_getProperty() throws Exception {
         assertThat(global.getAggregations(), notNullValue());
         assertThat(global.getAggregations().asMap().size(), equalTo(1));
 
-        GeoBounds geobounds = global.getAggregations().get(aggName);
+        SpatialBounds<GeoPoint> geobounds = global.getAggregations().get(aggName);
```

Comment:
The same here

Reply:
I've done this for the two dateline tests, but all the rest moved into shared code.